pytest_reportportal/plugin.py

```diff
@@ -92,13 +92,14 @@ def pytest_sessionstart(session):
 
         attributes = gen_attributes(
             session.config.getini('rp_launch_attributes'))
-        session.config.py_test_service.start_launch(
-            session.config.option.rp_launch,
-            attributes=attributes,
-            description=session.config.option.rp_launch_description
-        )
-        if session.config.pluginmanager.hasplugin('xdist'):
-            wait_launch(session.config.py_test_service.rp)
+        if not session.config.getini('rp_launch_id'):
```

Comment:
It's better to track this condition in the client rather than implementing the same "if" statements in every agent.